data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+			`	Teeneep sighs, "Captain <last>, I need you to escort me to <destination>." Stephen looks worried at that suggestion. "The Unfettered Hai have plenty of Jump Drives and troops. They could liberate our people. I know their ways, and I can convince them to help, if my ship survives the journey. I'll pay you double this time: <payment>."`
+				accept
+	npc save accompany
+		government " Hai "
+		personality timid escort
+		ship "Centipede (Teeneep)" "Teeneep's Ship"
+	on complete
+		payment 800000
+
+########################################################################
+
+# Part 3: Unfettered save the day, and everyone lives happily ever after
+
+mission "Hai Rescue [C01] Bloody Love Blooms"
+	landing
+	name "Missing Researchers"
+	description "Fly to <stopovers> to talk to the Hai authorities and see what can be done to arrange the requested outfitting. They will be waiting on <destination> when you finish."
+	source "Darkcloak"
+	stopover "Hai-home"
+	destination "Allhome"
+	to offer
+		has "Hai Rescue [B08] Re-Plan: done"
+	on offer
+		"reputation: Hai (Unfettered)" >?= 10
+		log `Teeneep has convinced the Unfettered Hai leader to send fifteen Shield Beetles with Jump Drives to human space for upgrades. They'll pass through the wormhole at Devil-Run and help you conquer the Philosopher's Paradise.`
+		conversation
+			`You meet Teeneep outside her Centipede and you're surprised to see her with no bodyguards. Instead of her Hai Elder robes, she's wearing what looks like a fencing outfit. There are two swords on her back and several balanced knives on her belt. She's wearing a headband with a medallion on it that looks like an athletic medal.`
+			`	She pays you for the trip and says, "These blades are not ceremonial. Leave your gun behind or the Unfettered may think you're an assassin. I'll protect you."`
+			`	You hand your gun to a nearby Unfettered, and hope Teeneep is as capable as she claims.`
+			`	Now that you see her in this guise, you remember a fencing match advertisement. She looks exactly like the fencing champion a few years ago. Is Teeneep a champion fencer as well as a civil rights leader and Hai Elder? From appearances, she hasn't been in adulthood for long. Though with such celebrity power, it's no surprise she rose in society quickly.`
+			`	"Let's go," she says.`
+			choice
+				`	"What is your plan?"`
+				`	(Follow her.)`
+					goto follow
+			`	"To Unfettered Hai leadership, this meeting is all a political ploy. They know it's in their best interest to help us. It will raise public opinion towards them among the Fettered and likely lure more recruits to their side. They cannot be seen as helping a weak coward though. I have to prove myself in front of the troops."`
```

Comment:
Just a thought, it seems odd that Teeneep would refer to her own people as the Fettered Hai, as it would confirm the unfettered conspiracy theory

Reply:
Upon further reading, scratch this comment